README.md

```diff
@@ -292,6 +292,7 @@ To add dependencies that are required for your analysis to the project Docker im
   * Installing most packages, from CRAN or Bioconductor, should be done  with our `install_bioc.R` script, which will ensure that the proper MRAN snapshot is used. `BiocManager::install()` should *not* be used, as it will not install from MRAN.
   * R packages that are not available in the MRAN snapshot can be installed via github with the `remotes::install_github()` function, with the commit specified by the `ref` argument.
 * Python packages should be installed with `pip3 install` with version numbers for all packages and dependencies specified.
+  * As a secondary check, we maintain a `requirements.txt` file to check versions of all python packages and dependencies. 
```

Comment:
What is someone supposed to do if this fails? Their knee-jerk reaction could be to roll forward `requirements.txt`, which is not necessarily what we want them to do.

Reply:
My thought was that this will cause some discussion & be evident in the review. The idea is really to make sure that all added packages are in both the Docker & requirements file, mostly because it is easy to add a python package and not realize that it is also adding a dependency that may not be versioned. So a reviewer would hopefully catch the discrepancy... Also, if rolling forward requirements.txt results in all tests passing, maybe that is okay?

Reply:
We are on the same page, but I was wondering if we wanted to note that the best course of action for updating requirements will need to be evaluated on a case-by-case basis.

Reply:
I added a bit more to the comment here: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1008/commits/4d3fc461da65bd830ade2101cf415ff1275f7453
I hope that we do not have any trouble with this at this stage, but maybe I am too optimistic.

Reply:
Works for me!